system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -337,20 +337,69 @@ $half-base-gutter: $base-gutter / 2;
         transform: translateY(-50%);
         top: 50%;
         offset-inline-end: 8px;
+      }
+    }
 
-        & + input:dir(ltr) {
-          padding-right: 32px;
-        }
+    .url {
+      input:dir(ltr) {
+        padding-right: 32px;
+      }
 
-        & + input:dir(rtl) {
-          padding-left: 32px;
+      input:dir(rtl) {
+        padding-left: 32px;
+
+        &:not(:placeholder-shown) {
+          direction: ltr;
+          text-align: right;
         }
       }
     }
 
-    .url input:not(:placeholder-shown):dir(rtl) {
-      direction: ltr;
-      text-align: right;
+    .enable-custom-image-input {
+      display: inline-block;
+      font-size: 13px;
+      margin-top: 4px;
+
+      &:hover {
+        text-decoration: underline;
+        color: $blue-60;
+      }
+    }
+
+    .custom-image-input-container {
+      margin-top: 4px;
+
+      .loading-container {
+        width: 16px;
+        height: 16px;
+        overflow: hidden;
+        position: absolute;
+        transform: translateY(-50%);
+        top: 50%;
+        offset-inline-end: 8px;
+      }
+
+      // See Bug 1352119
```

Comment:
This comment isn't particularly useful as is. What's the bug for? Is it to fix something undesired here?